dist/index.html

```diff
@@ -1 +1,43 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>DOC</title><script defer="defer" src="main.js"></script></head><body><h1>TODO list!!</h1><div class="list-container"></div></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,300;1,400;1,600;1,700;1,800&display=swap" rel="stylesheet">
+    <link
+			rel="stylesheet"
+			href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+			integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+			crossorigin="anonymous"
+		/>
+    <title>DOTO</title>
+<script defer src="main.js"></script></head>
+<body>  
+    <div class="container">
+        <header class='header'>
+            <h1>DO::TO::list</h1>
```

Comment:
- [OPTIONAL] I think this header could improve for margin/padding and what it said. If you can I invite you to remove it or change the margin/padding and text.
![image](https://user-images.githubusercontent.com/20563252/125901255-ce0b8459-cfac-4952-b88e-83a0ec5bbff1.png)